src/Components/LoginComponent/Signup.js

```diff
@@ -0,0 +1,319 @@
+import React, { useState } from "react";
+import styled, { ThemeProvider } from "styled-components";
+import { withRouter } from "react-router-dom";
+import Apiresource from "../../Config";
+
+
+function Signup(props, visible) {  
```

Comment:
```suggestion
function Signup(props) {  
```
visible이라는 변수가 인자로 들어오는건가요..??

Reply:
visible을 빼버리면 모달의 배경이 검정색이 없어져서 다시 넣었고, 빈객체이나
넣은 이유가 인터넷에서 찾아보니 작성시 필요하다고 해서 집어넣었습니다.
현재 display: prop.visible로 사용중입니다.